### ISSUE TITLE
[MU4] Reopening Style does no longer recall the tab last used

### ISSUE THREAD

**Issue by oMrSmith** [musescore/MuseScore#13242]
In MU3 opening Style always recalls the tab last used, which I find very helpful.

This does no longer happen with MU4.

Not sure if this is intended...

**Comment by MarcSabatella** [comment]
Right now this is consistent with other windows such as Preferences and the Instruments list.  All three of these remembered state in MU4, and I think they all three should in MU4 as well.

A potential complicating factor is the fact that there are potentially multiple instances of MuseScore open at once, and probably we don't want actions in one instance affecting the others.  But as long as the state is maintained in memory rather than by INI files, that should work as expected.

**Comment by oMrSmith** [comment]
(It doesn't work however.)

**Comment by Tantacrul** [comment]
Yeah. Agree it should remember the last tab you were on within the same session.

When in a new session, it should reset.
